text/000-rustdoc-jump-to-definition.md

```diff
@@ -121,14 +121,19 @@ Anything not specifically mentioned should not get a link generated for itself.
 
 #### Local variables
 
-Local variables should not get a link:
+Local variables should get a link:
```

Comment:
Should be clear that you mean a link to their definition (not their type) here.

Reply:
Oh indeed! Thanks!